cinema/tests/test_movie_session_api.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+
+from django.test import TestCase
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+from cinema.models import Movie, Genre, Actor, MovieSession, CinemaHall
+
+
+class MovieSessionApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        drama = Genre.objects.create(
+            name="Drama",
+        )
+        comedy = Genre.objects.create(
+            name="Comedy",
+        )
+        actress = Actor.objects.create(first_name="Kate", last_name="Winslet")
+        movie = Movie.objects.create(
+            title="Titanic",
+            description="Titanic description",
+        )
+        movie.genres.add(drama)
+        movie.genres.add(comedy)
+        movie.actors.add(actress)
+        cinema_hall = CinemaHall.objects.create(
+            name="White",
+            rows=10,
+            seats_in_row=14,
+        )
+        MovieSession.objects.create(
+            movie=movie, cinema_hall=cinema_hall, show_time=datetime.datetime.now()
+        )
+
+    def test_get_movies(self):
+        movie_sessions = self.client.get("/api/cinema/movie_sessions/")
+        movie_session = {
+            "movie_title": "Titanic",
+            "cinema_hall_name": "White",
+            "cinema_hall_capacity": 140,
```

Comment:
So right here for ex. you will be able to write not hard-coded values, but `self.movie.title`